docs/modules/ROOT/pages/reference/glossary.adoc

```diff
@@ -0,0 +1,62 @@
+= Glossary
+
+== General
+
+=== Service
+A managed application that can be ordered, is auomatically provisioned and can be consumed by a service consumer.
+
+=== Service Consumer
+A customer-owned application that connects to a service as a client.
```

Comment:
```suggestion
An application that consumes a service instance.
```

I wouldn't focus on "customer", it can be anyone, even another system component.

Reply:
Agree, but I wouldn't use `consume` again, since that is the term we want to define ;)